### DISCUSSION TITLE
Add ETH to a contractWrite hook transaction

### DISCUSSION THREAD

**Discussion by raffle-x-dev** [wevm/wagmi#3560] [Q&A]
I need to send ETH along with some ERC20 token transfer.
I'musing useContractRead hook to call the solidity function that performs the transfer. How can I add ETH value to the transaction in the frontend?

**Comment by POKENA7** [comment]
@raffle-x-dev 
it might help
https://1.x.wagmi.sh/react/hooks/useContractWrite#value-optional
